package.json

```diff
@@ -20,7 +20,7 @@
   },
   "homepage": "https://github.com/ksachdeva/angular2-swing#readme",
   "dependencies": {
-    "swing": "^4.3.0"
+    "swing": "^3.1.0"
```

Comment:
3.1.0 is the latest version from what I can tell. https://www.npmjs.com/package/swing. Not sure why 4.3.0 was in there.